custom/icds_reports/models/util.py

```diff
@@ -164,14 +164,14 @@ def setup_days_records(cls, day):
                 ]
             },
             cls.CommCareCase: {
-                'commcare-user': ['static-commcare_user_cases'],
-                'ccs_record': ['static-ccs_record_cases'],
-                'child_health': ['static-child_health_cases'],
-                'hardware': ['static-hardware_cases'],
-                'household': ['static-household_cases'],
-                'person': ['static-person_cases_v3'],
-                'tasks': ['static-tasks_cases'],
-                'tech_issue': ['static-tech_issue_cases'],
+                'static-commcare_user_cases': ['commcare-user'],
+                'static-ccs_record_cases': ['ccs_record'],
+                'static-child_health_cases': ['child_health'],
+                'static-hardware_cases': ['hardware'],
+                'static-household_cases': ['household'],
+                'static-person_cases_v3': ['person'],
+                'static-tasks_cases': ['tasks'],
+                'static-tech_issue_cases': ['tech_issue'],
```

Comment:
I already fixed these in the models already created and will fix the other ones as they are created